packages/manager/apps/veeam-backup/src/pages/order-veeam/OrderVeeamStep2.component.tsx

```diff
@@ -96,11 +96,8 @@ export const OrderVeeamStep2: React.FC = () => {
     <>
       <Title className="block mb-9">{t('choose_org_title')}</Title>
       {isLoading && <Loading className="mb-5" />}
-      {!isLoading && data?.pages[0].data.length === 0 && (
-        <OsdsMessage type={ODS_MESSAGE_TYPE.warning}>
-          {t('no_organization_message')}
-          <VCDOrgInfoLink label={t('no_organization_link')} />
-        </OsdsMessage>
+      {!isLoading && (
+        <NoOrganizationMessage organizationList={data?.pages[0].data} />
```

Comment:
If you have an error. You have not JS error ?

I don't known `data?.pages[0]` is defined

Reply:
data is undefined if there is an error but I added a check for a isError
Thanks